src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20200501145338_EES-669_Remove_OriginalId_Column.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Migrations.ContentMigrations
+{
+    public partial class EES669_Remove_OriginalId_Column : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_Releases_Releases_OriginalId",
+                table: "Releases");
+
+            migrationBuilder.DropIndex(
+                name: "IX_Releases_OriginalId_Version",
+                table: "Releases");
+
+            migrationBuilder.DropColumn(
+                name: "OriginalId",
+                table: "Releases");
+
+            migrationBuilder.UpdateData(
+                table: "Releases",
+                keyColumn: "Id",
+                keyValue: new Guid("4fa4fe8e-9a15-46bb-823f-49bf8e0cdec5"),
+                column: "PreviousVersionId",
+                value: new Guid("4fa4fe8e-9a15-46bb-823f-49bf8e0cdec5"));
```

Comment:
Just wondering if these 3 updates have been covered by the custom SQL statement running in the previous migration.
`SET PreviousVersionId = OriginalId`?

If it's only doing the same thing I suppose it's no big issue.

Reply:
True but it would need to present in the "down" I think so probably clearer to leave.